### PR TITLE
fix: Early Access Feature Opt-In/Out Search Not Working

### PR DESCRIPTION
**Pull request by nategrift**
## Problem
Pt 2 of https://github.com/PostHog/posthog/issues/16946 (they are separate bugs but same component)
- Within the Early Access Feature menu, the opt-in/opt-out search was not working. Typing in the box would not change anything with the list down below. 

**Technical Issue:**
The `PersonsSearch` and `PersonsTable` kea logic was not synced. They were using separate instances of the `personsLogic`.

## Changes

-To solve this issue, we created a wrapper which binds them to all use the same logic. We want to have separate logics for both the `Opt-In` and `Opt-Out` tabs as they have different properties. 

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
- Tested Locally

### CODE REVIEW

**Comment by nategrift** [comment]
@mariusandra Are you able to review this or assign this PR to the correct person? Thanks!

File: frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx [hunk 1]
```diff
@@ -387,17 +387,29 @@ interface PersonsTableByFilterProps {
...
-    const { loadPersons, setListFilters } = useActions(logic)
+
+    return (
+        <BindLogic logic={personsLogic} props={personsLogicProps}>
+            <PersonsTableByFilterScene emptyState={emptyState} />
+        </BindLogic>
+    )
+}
+
+interface PersonsTableByFilterSceneProps {
```
Comment (neilkakkar) [inline frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx:404]: hey, thanks Nate, this looks good, except the name `Scene`, which is reserved for standalone components that make up the whole page, I'd rather call this `Component` instead.

**Comment by neilkakkar** [comment]
Hey @nategrift mind merging in master? I think this CI issue was fixed recently, can get it in once tests pass :)

**Comment by nategrift** [comment]
Merged master in @neilkakkar, hopefully the tests pass now!

**Comment by neilkakkar** [comment]
lovely, thanks!

**Pull request merged**